src/main/java/study/StringAddCalculator.java

```diff
@@ -0,0 +1,58 @@
+package study;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public int splitAndSum(String value)throws Exception{
+        int result = checkNull_Empty(value);
+        String[] values;
+
+        if(result == 0){
+            return result;
+        }else{
+            values = checkSeparator(value);
+            int calcul = 0;
+            for(String val : values){
+                calcul += checkNegative(Integer.valueOf(val));
+            }
+            result = calcul;
+        }
```

Comment:
```suggestion
        if (result == 0) {
            return result;
        } else {
            values = checkSeparator(value);
            int calcul = 0;
            for (String val : values) {
                calcul += checkNegative(Integer.valueOf(val));
            }
            result = calcul;
        }
```
들여쓰기와 공백 문자를 적절히 사용하면 가독성을 향상 시킬수 있어요. 
Intellj를 사용하고 계시다면 아래 가이드를 통해 코드 포맷을 맞춰보세요 😄 
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html